src/priyanka_Panat/LoopExample.java

```diff
@@ -0,0 +1,67 @@
+package priyanka_Panat;
+
+/*
+ Assignment 8
+ */
+
+class LoopExample{
+	void evenNum(int startindex,int endindex){
+		int i=10;
```

Comment:
you don’t require to create extra variable, you can handle this without **i**, think on this
you haven’t use startIndex variable anywhere

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/219/files